src/mfs.jl

```diff
@@ -0,0 +1,230 @@
+using BlackBoxOptim: bboptimize, best_candidate
```

Comment:
You should re-organize this file following good practices in scientific code. This means: you start with the main functions, and after the main functions and types you write all the other functions that are called. I describe this in the good scientific code workshop, block 2, slide 21: https://github.com/JuliaDynamics/GoodScientificCodeWorkshop/blob/v1.0/block2_clearcode/block2_slides.pdf 

this means, that the more inner and more small detail a function is, the later it comes in the file. Makes for a better reading experience.